docs/modules/setup/pages/use-kubernetes.adoc

```diff
@@ -163,3 +166,67 @@ unzip kroki-k8s-singlepod.zip -d kroki-k8s
 You can then create the Kubernetes objects as a batch or individually, similar to what was shown previously.
 
 Validation and uninstall/removal of Kubernetes objects can be followed using the exact same commands as shown previously too.
+
+== Deploy using Helm Charts
+You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by helm.
```

Comment:
```suggestion
You can alternatively deploy Kroki using {uri-helm-charts}[Helm Charts]. It's much easier to deploy Kroki with charts, since all object definition files are managed by Helm.
```